tests/failing/describeAndContext.spec.lua

```diff
@@ -0,0 +1,7 @@
+-- luacheck: globals describe
+
+return function()
+	describe("this shouldn't be able to access context", function(context)
+		context.foo = "bar"
```

Comment:
What kind of error should this be? Is `context` nil, or is it just that we can't write to it in a `describe` block?

Reply:
Context is nil. (There wouldn't be anything to read if it wasn't.) I could make that an explicit check, though the test I really want is "the first argument isn't a context object" in case someone adds a parameter to describe later, but maybe that's overthinking things?